packages/driver/src/cli.mts

```diff
@@ -95,46 +95,48 @@ async function main(args: string[]) {
 async function whichEdgeDbCli() {
   debug("Checking if CLI is in PATH...");
   const locations = await which("edgedb", { nothrow: true, all: true });
-  for (const location of locations) {
-    const actualLocation = await fs.realpath(location);
-    debug(
-      `  - CLI found in PATH at: ${location} (resolved to: ${actualLocation})`,
-    );
+  if (locations) {
```

Comment:
Small nit: maybe just treat this as an early return instead of wrapping this whole section in an `if`?

```
if (locations == null) {
  debug("  - No CLI found in PATH.");
  return null;
}
```

I'm annoyed that we aren't catching this in our tests 🫤